src/overlays/actors/ovl_En_Fu/z_en_fu.h

```diff
@@ -2,6 +2,7 @@
 #define Z_EN_FU_H
 
 #include "global.h"
+#include "overlays/actors/ovl_Bg_Fu_Mizu/z_bg_fu_mizu.h"
```

Comment:
Because it's used in this header. You commented this on another PR and it's the same issue. Inclusing this header throws errors because BgFuMizu is undefined.